src/ButtonV2/ButtonV2.stories.tsx

```diff
@@ -17,3 +17,12 @@ Controlled.args = {
   radius: 'standard',
   children: 'This is my button',
 };
+
+export const ControlledNew = Template.bind({});
+ControlledNew.args = {
```

Comment:
When the variant has been updated can you change this to be Round{VariantName} or something

Reply:
I've removed this story (see my previous comment).